packages/lit-dev-content/site/docs/localization/overview.md

```diff
@@ -444,7 +498,7 @@ Descriptions will be represented in XLIFF files using `<note>` elements.
 ## Message ids
```

Comment:
I didn't notice this before, but would probably be better to use ID in text. (Except where referring to a literal property or in XLIFF markup.)  https://developers.google.com/style/word-list#id

Seems like it's more likely to read as  "eye-dee" like your driver's license rather than "id," the opposite of the superego.

Reply:
Done